src/diagnosticMessages.json

```
@@ -12,6 +12,7 @@
   "Start function name '{0}' is invalid or conflicts with another export.": 110,
   "Element '{0}' not found.": 111,
   "Exchange of '{0}' values is not supported by all embeddings": 112,
+  "Index signature type should be the same for element accessors in type '{0}'.": 113,
```

```suggestion
  "Index signature in type '{0}' differs in types.": 113,
```
...then with errorRelated showing both places? :)